diff --git a/app/include/user_config.h b/app/include/user_config.h
index c786e90..61ce359 100644
--- a/app/include/user_config.h
+++ b/app/include/user_config.h
@@ -110,7 +110,7 @@ extern void luaL_assertfail(const char *file, int line, const char *message);
 //#define WIFI_STA_HOSTNAME "NodeMCU"
 //#define WIFI_STA_HOSTNAME_APPEND_MAC
 
-//#define WIFI_SMART_ENABLE
+#define WIFI_SMART_ENABLE
 
 #define WIFI_STATION_STATUS_MONITOR_ENABLE
 #define WIFI_SDK_EVENT_MONITOR_ENABLE
diff --git a/app/include/user_modules.h b/app/include/user_modules.h
index cf4f5d5..0b8d04d 100644
--- a/app/include/user_modules.h
+++ b/app/include/user_modules.h
@@ -27,7 +27,7 @@
 //#define LUA_USE_MODULES_CJSON
 //#define LUA_USE_MODULES_COAP
 //#define LUA_USE_MODULES_CRON
-//#define LUA_USE_MODULES_CRYPTO
+#define LUA_USE_MODULES_CRYPTO
 #define LUA_USE_MODULES_DHT
 //#define LUA_USE_MODULES_ENCODER
 //#define LUA_USE_MODULES_ENDUSER_SETUP // USE_DNS in dhcpserver.h needs to be enabled for this module to work.
@@ -35,7 +35,7 @@
 //#define LUA_USE_MODULES_GDBSTUB
 #define LUA_USE_MODULES_GPIO
 //#define LUA_USE_MODULES_HMC5883L
-//#define LUA_USE_MODULES_HTTP
+#define LUA_USE_MODULES_HTTP
 //#define LUA_USE_MODULES_HX711
 #define LUA_USE_MODULES_I2C
 //#define LUA_USE_MODULES_L3G4200D
@@ -46,21 +46,21 @@
 #define LUA_USE_MODULES_OW
 //#define LUA_USE_MODULES_PCM
 //#define LUA_USE_MODULES_PERF
-//#define LUA_USE_MODULES_PWM
+#define LUA_USE_MODULES_PWM
 //#define LUA_USE_MODULES_RC
 //#define LUA_USE_MODULES_RFSWITCH
 //#define LUA_USE_MODULES_ROTARY
 //#define LUA_USE_MODULES_RTCFIFO
 //#define LUA_USE_MODULES_RTCMEM
-//#define LUA_USE_MODULES_RTCTIME
+#define LUA_USE_MODULES_RTCTIME
 //#define LUA_USE_MODULES_SIGMA_DELTA
-//#define LUA_USE_MODULES_SNTP
+#define LUA_USE_MODULES_SNTP
 //#define LUA_USE_MODULES_SOMFY
 #define LUA_USE_MODULES_SPI
 //#define LUA_USE_MODULES_STRUCT
 //#define LUA_USE_MODULES_SWITEC
 //#define LUA_USE_MODULES_TM1829
-#define LUA_USE_MODULES_TLS
+//#define LUA_USE_MODULES_TLS
 #define LUA_USE_MODULES_TMR
 //#define LUA_USE_MODULES_TSL2561
 //#define LUA_USE_MODULES_U8G
@@ -70,7 +70,7 @@
 #define LUA_USE_MODULES_WIFI
 //#define LUA_USE_MODULES_WPS
 //#define LUA_USE_MODULES_WS2801
-//#define LUA_USE_MODULES_WS2812
+#define LUA_USE_MODULES_WS2812
 
 #endif  /* LUA_CROSS_COMPILER */
 #endif	/* __USER_MODULES_H__ */
